eclipse/che-devfile-registry
# Add realworld go sample

Signed-off-by: Anatoliy Bazko <abazko@redhat.com>



### What does this PR do?
It adds real-world-go-sample
https://github.com/xesina/golang-echo-realworld-example-app

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13803

### Related PRs
https://github.com/eclipse/che-plugin-registry/pull/224
https://github.com/eclipse/che-theia/pull/443

### How to test
1. replace plugin id to with the reference
```
id: ms-vscode/go/latest
```
-> 
```
 reference: >-
      https://raw.githubusercontent.com/eclipse/che-plugin-registry/11ea1728310fd8ed6b26e904d4652d7aa14f166b/v3/plugins/ms-vscode/go/0.11.4/meta.yaml
```

2. Build `docker.io/eclipse/che-remote-plugin-go-1.12.9:next` (see https://github.com/eclipse/che-theia/pull/443)
or retag it from `docker.io/abazko/che-remote-plugin-go-1.12.9:next`
